qemu/tests/cfg/block_kill_reconnect_with_remote_storage.cfg

```diff
@@ -0,0 +1,36 @@
+# Network storage backends:
+#   nbd
+# The following testing scenarios are covered:
+#   kill/reconnect a remote virtio-scsi image
+#   kill/reconnect a remote virtio-blk image
+
+- block_kill_reconnect_with_remote_storage:
+    no RHEL.3.9
+    only nbd
+    virt_test_type = qemu
+    type = block_kill_reconnect_with_remote_storage
+    bootindex_image1 = 0
+    images += " data1"
+    boot_drive_data1 = yes
+    image_size_data1 = 1G
+    nbd:
+        nbd_port_data1 = 10822
+        force_create_image_data1 = no
+        image_create_support_data1 = no
+        remove_image_data1 = no
+    get_disk_cmd = "ls /dev/[hsv]d[a-z]* | sort"
+    disk_op_cmd = "dd if=%s of=/dev/null bs=1k count=1000 iflag=direct &&"
+    disk_op_cmd += " dd if=/dev/zero of=%s bs=1k count=1000 oflag=direct"
```

Comment:
This may be not a good idea, when we dd data to system image, we might damage it.

Reply:
Changes to dd if=/dev/zero of=test_file bs=1k count=1000 oflag=direct && dd if=test_file of=/dev/null bs=1k count=1000 iflag=direct